src/lab10.py

```diff
@@ -160,7 +160,7 @@ def load(self, url, payload=None):
             except dukpy.JSRuntimeError as e:
                 print("Script", script, "crashed", e)
 
-        self.rules = self.default_style_sheet.copy()
+        self.rules = DEFAULT_STYLE_SHEET.copy()
```

Comment:
Did lab10 just not work previously?

Reply:
I think `lab10` was using the `default_style_sheet` field being created in `lab8`. So yes `lab8` was redundantly storing the default style sheet in _both_ places, and some labs were using one place while some were using the other.